fxos-dev-mode-dialog.js

```diff
@@ -15,7 +15,7 @@ var template = `<style>
 .modal-dialog > section {
   color: #FAFAFA;
   padding: 80px 25px 0;
-  -moz-box-sizing: padding-box;
+  -moz-box-sizing: border-box;
   font-size: 22px;
```

Comment:
Looks good, we should update bower.json and package.json file in fxos-dev-mode-dialog to version 0.0.4, so that this change can be picked by apps using fxos-dev-model-dialog